spec/display-buffer-spec.coffee

```diff
@@ -1272,3 +1272,34 @@ describe "DisplayBuffer", ->
       displayBuffer.setScrollTop(60)
 
       expect(displayBuffer.getVisibleRowRange()).toEqual [6, 13]
+
+  describe "::decorateMarker", ->
+    describe "translates gutter decorations to distinguish custom gutters from the line-number gutter", ->
+      [marker] = []
+
+      beforeEach ->
+        marker = displayBuffer.markBufferRange([[1, 0], [1, 0]]);
+
+      it 'translates an unnamed gutter to the line-numbers gutter', ->
```

Comment:
assigns the decoration to the line-numbers gutter if type is "gutter" and there is no gutter name